main.py

```diff
@@ -2,12 +2,30 @@
 import pyautogui
 import sys
 import uvicorn
-from fastapi import FastAPI
+from fastapi import FastAPI, WebSocket
 from fastapi.staticfiles import StaticFiles
 from fastapi.responses import FileResponse
 
 app = FastAPI()
+pyautogui.FAILSAFE = False
+pyautogui.PAUSE = 0
 
+@app.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
+    print('Accepting client connections..')
+    await websocket.accept()
+
+    while True:
+        try:
+            data = await websocket.receive_json()
+            if data['type'] == "move":
+                pyautogui.moveRel(data['x'], data['y'])
+            elif data['type'] == "tap":
+                pyautogui.leftClick()
+        except Exception as ex:
```

Comment:
Let's catch specific exceptions. In this case, it could be `WebSocketDisconnect`